uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-calendar-legend.component.spec.js

```diff
@@ -29,6 +29,15 @@ define([
       expect(controller.legendCollapsed).toBe(true);
     });
 
+    it('has a list of "other" badges', function () {
```

Comment:
We should have a single expect statement in each `it` block and nest them under a `describe`.
Something like
`has a list of "other" badges`
`each other badge contains label and hint values`
`has a total of 5 other badges`

Reply:
Did it slightly differently, but hopefully still satisfied you request.